packages/cli/src/lifecycles/config.js

```diff
@@ -11,6 +11,7 @@ let defaultConfig = {
   publicPath: '/',
   title: 'Greenwood App',
   meta: [],
+  optimization: {},
```

Comment:
this should not be a public API (yet), we'll think of something but I want to keep this under the hood